src/book.js

```diff
@@ -392,9 +393,9 @@ export class Book {
         const back = config.back;
         let filteredList = [];
         let blankIndices = [];
-        pagelist.forEach((page, i) => {
-            if (page != 'b') {
-                filteredList.push(page);
+        pagelist.forEach((pageInfo, i) => {
+            if (pageInfo != 'b') {
```

Comment:
This seems to be ad odds with the param description - should it not be `pageInfo.info`?

Reply:
ACK! Good catch!! I really thought I had tested the blank pages with this 😓